mailbox/store/src/main/java/org/apache/james/mailbox/store/quota/ListeningCurrentQuotaUpdater.java

```diff
@@ -87,7 +90,7 @@ private void handleExpungedEvent(Expunged expunged, QuotaRoot quotaRoot) throws
         long expungedCount = (long) expunged.getUids().size();
         // Expunge event can contain no data (expunge performed while no messages marked \Deleted)
         if (expungedCount != 0 && expungedSize != 0) {
-            currentQuotaManager.decrease(quotaRoot, expungedCount, expungedSize);
+            currentQuotaManager.decrease(new CurrentQuota(quotaRoot, QuotaCountUsage.count(expungedCount), QuotaSizeUsage.size(expungedSize)));
         }
         eventBus.dispatch(
             EventFactory.quotaUpdated()
```

Comment:
We dispatch a quota updated event even when the quota is not updated?

Reply:
Well seems we do (I didn't change that part)... But yeah I guess should not. Will fix it